Eng 8092: Add HTTP Export Capability

pdshaw opened the pull request:
Added http export as a new run.sh option, includied simple webserver as sink for exported messages, and updated run.sh to eliminate "catalog" step.

Commit:
Add export table and enable in deployment.xml

Commit:
Export trigger added.

Commit:
Add card owner demographics to cards and to export.

Update card setup to stuff in owner info too.

Commit:
Randomize the notify settings in cards table initialization.

Commit:
Webserver side of the export demo

Commit:
Updates
    new endpoint name
    change export time type from timestamp string to epoch time in milliseconds

Commit:
Lower tps rate to reduce time out and reset exceptions due to overwelling the web service.

Commit:
Reduce percentage "opted in" for notify
Add refresh to static html page output by server

Commit:
Add copyright header

akhanzode commented:
You saw Ben's email. What we need to finish it is
1. Create a export related deployment.
2. Modify run to start with switch with export. Keep default without export
3. Modify any instructions for running export demo.

Commit:
Updates for interactive DDL.

Commit:
Back up to Master content. Adding export capabilities in a separate file.

Commit:
Version of CardSwipe.java with export capability.

Commit:
Split export off as separate deployment file, and related changes.

Commit:
More changes needed to complete split of export-enabled and "default"

Commit:
No need for separate export version.

pdshaw commented:
The README needs a little more work; the functionality is complete.

Commit:
"set -x" off
update usage list
further updates due to catalog elimination

Commit:
A little more cleanup of the run script.

Commit:
Tie everytogether --
   - runs.sh: additional launch functions for export web server in run.sh
   - exportServer.py: catch and handle exception if the Twilio module has not been installed
   - deployment-export.xml: switch my IP address to generic "localhost"
   - README.md: update instructions for addition functionality

benjaminballard committed:
minor corrections
